client/src/components/Overview/ProductInfo.jsx

```diff
@@ -21,107 +21,97 @@ const ProductInfo = () => {
   const productInfo = state.products.filter((product) => product.id === state.selectedProduct)[0];
   const styleInfo = state.styles.filter((style) => style.style_id === state.selectedStyle)[0];
```

Comment:
I super forgot about the filter function. I'm may refactor my code later to use this and speed up optimization for my components